### ISSUE TITLE
Issue downloading EPROM data

### ISSUE THREAD

**Issue by msp1974** [msp1974/visonic_proxy#3]
The integration starts to download the EPROM data and then stops after getting the panel info message.  I have had it download a few times but can't see why it does sometimes and not others.  Mainly not.
```
2024-08-02 00:25:11.042 INFO     -------------------------------------------------------------------------------------------
2024-08-02 00:25:11.042 INFO     CONNECTIONS: Alarm: ['P37457'], Visonic: [], HA: []
2024-08-02 00:25:11.042 INFO     MODES: Disconnected Mode: True, Stealth Mode: False
2024-08-02 00:25:11.042 INFO     -------------------------------------------------------------------------------------------
2024-08-02 00:25:20.285 INFO     Client connected to HA P40220 server. Clients: 1
2024-08-02 00:25:20.285 INFO     ConnectionManager->HA P40220-0000 VIS-BBA 0d e0 01 00 01 00 43 d9 0a
2024-08-02 00:25:20.287 INFO     HA->Alarm P37457-4303 VIS-BBA 0d 0b f4 0a
2024-08-02 00:25:21.043 INFO     -------------------------------------------------------------------------------------------
2024-08-02 00:25:21.043 INFO     CONNECTIONS: Alarm: ['P37457'], Visonic: [], HA: ['P40220']
2024-08-02 00:25:21.043 INFO     MODES: Disconnected Mode: True, Stealth Mode: False
2024-08-02 00:25:21.043 INFO     -------------------------------------------------------------------------------------------
2024-08-02 00:25:23.291 INFO     Entering Stealth Mode
2024-08-02 00:25:23.291 WARNING  Timeout waiting for ACK from Alarm for msg id 4303
2024-08-02 00:25:23.292 INFO     HA->Alarm P37457-4303 VIS-BBA 0d 0f f0 0a
2024-08-02 00:25:26.295 WARNING  Timeout waiting for ACK from Alarm for msg id 4303
2024-08-02 00:25:26.295 INFO     HA->Alarm P37457-4303 VIS-BBA 0d 09 f6 0a
2024-08-02 00:25:27.522 INFO     Alarm->HA P40220-4303 VIS-ACK 0d 02 fd 0a
2024-08-02 00:25:27.682 WARNING  ACK received with no destination. Event(name=<ConnectionName.ALARM: 'Alarm'>, event_type=<EventType.DATA_RECEIVED: 'data_received'>, client_id='P37457', event_data=PowerLink31Message(crc16='13D2', length='0021', msg_type='VIS-ACK', msg_id=4304, account_id='001234', panel_id='XXXXXX', message_class='02', data=b'\r\x02\xfd\n', raw_data=b'\n13D20021"VIS-ACK"4304L001234#25594E[\r\x02\xfd\n]\r'), destination=None, destination_client_id=None)
2024-08-02 00:25:27.692 INFO     Alarm->HA P40220-4305 VIS-BBA 0d 3c fd 0a 00 00 ae 07 01 00 00 00 05 0a
2024-08-02 00:25:27.693 INFO     HA->Alarm P37457-4305 VIS-ACK 0d 02 fd 0a
2024-08-02 00:25:27.845 INFO     HA->Alarm P37457-4306 VIS-BBA 0d 24 00 00 aa aa 00 00 00 00 00 00 86 0a
2024-08-02 00:25:27.875 INFO     Alarm->HA P40220-4306 VIS-ACK 0d 02 fd 0a
2024-08-02 00:25:27.919 INFO     Alarm->HA P40220-4307 VIS-BBA 0d 3c fd 0a 00 00 ae 07 01 07 00 00 fd 0a
2024-08-02 00:25:27.996 INFO     HA->Alarm P37457-4307 VIS-ACK 0d 02 fd 0a
2024-08-02 00:25:28.582 INFO     Alarm->HA P40220-4308 VIS-BBA 0d b0 03 51 06 ff 08 ff 01 24 a7 43 dc 0a
2024-08-02 00:25:28.582 INFO     ConnectionManager->Alarm P37457-4308 VIS-ACK 0d 02 fd 0a
2024-08-02 00:25:31.047 INFO     -------------------------------------------------------------------------------------------
2024-08-02 00:25:31.048 INFO     CONNECTIONS: Alarm: ['P37457'], Visonic: [], HA: ['P40220']
2024-08-02 00:25:31.048 INFO     MODES: Disconnected Mode: True, Stealth Mode: True
2024-08-02 00:25:31.048 INFO     -------------------------------------------------------------------------------------------
2024-08-02 00:25:41.057 INFO     -------------------------------------------------------------------------------------------
2024-08-02 00:25:41.057 INFO     CONNECTIONS: Alarm: ['P37457'], Visonic: [], HA: ['P40220']
2024-08-02 00:25:41.057 INFO     MODES: Disconnected Mode: True, Stealth Mode: True
2024-08-02 00:25:41.057 INFO     -------------------------------------------------------------------------------------------

```

**Comment by msp1974** [comment]
This is the debug log
```
2024-08-02 00:25:20.280 DEBUG (ImportExecutor_0) [custom_components.visonic.pyvisonic] Initialising Protocol - Protocol Version 1.3.5.2
2024-08-02 00:25:20.285 DEBUG (MainThread) [custom_components.visonic.pyvisonic] Establishing Protocol - Using Home Assistant Loop
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Settings] Force Standard set to False
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Settings] Disable All Commands set to False
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Settings] Download Code set to aa aa
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Settings] Language set to EN
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Settings] Siren Trigger List set to ['intruder']
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Settings] ForceStandard to False     DisableAllCommands to False
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Connection] Connected to local Protocol handler and Transport Layer
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [_clearList] Setting queue empty
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [StartDownload] Triggering EEPROM download sequence
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [sendPdu] Sending Command (Stop)    waiting for message response []
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [sendPdu]          Command has a wait time after transmission 1.5
2024-08-02 00:25:20.286 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Controller] download_counter is 1
2024-08-02 00:25:21.288 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Controller] download_counter is 2
2024-08-02 00:25:21.789 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [sendPdu] Sending Command (Exit)    waiting for message response []
2024-08-02 00:25:21.789 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [sendPdu]          Command has a wait time after transmission 1.5
2024-08-02 00:25:22.292 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Controller] download_counter is 3
2024-08-02 00:25:23.291 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [sendPdu] Sending Command (Bump)    waiting for message response ['0X3C']
2024-08-02 00:25:23.291 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [sendPdu]          Command has a wait time after transmission 0.2
2024-08-02 00:25:23.293 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Controller] download_counter is 4
2024-08-02 00:25:24.295 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Controller] download_counter is 5
2024-08-02 00:25:25.297 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Controller] download_counter is 6
2024-08-02 00:25:26.298 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Controller] download_counter is 7
2024-08-02 00:25:27.306 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Controller] download_counter is 8
2024-08-02 00:25:27.692 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [handle_msgtype3C] Packet = fd 0a 00 00 ae 07 01 00 00 00
2024-08-02 00:25:27.692 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [handle_msgtype3C] Using the user defined Download Code <obfuscated>
2024-08-02 00:25:27.692 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [handle_msgtype3C] PanelType=7 : PowerMaster 10 , Model=174   Powermaster True
2024-08-02 00:25:27.692 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [handle_msgtype3C] interval=0:02:47.406649  td=0:01:00   self.lastSendOfDownloadEprom(UTC)=2024-08-01 23:22:40.286148    timenow(UTC)=2024-08-01 23:25:27.692805
2024-08-02 00:25:27.692 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [Panel Settings] Downloading panel settings
2024-08-02 00:25:27.845 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [_clearList] Setting queue empty
2024-08-02 00:25:27.845 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [sendPdu] Setting Download Mode to true
2024-08-02 00:25:27.845 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [sendPdu] Sending Command (Start Download Mode)    waiting for message response []
2024-08-02 00:25:27.920 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [handle_msgtype3C] Packet = fd 0a 00 00 ae 07 01 07 00 00
2024-08-02 00:25:27.920 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [handle_msgtype3C] Using the user defined Download Code <obfuscated>
2024-08-02 00:25:27.920 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [handle_msgtype3C] PanelType=7 : PowerMaster 10 , Model=174   Powermaster True
2024-08-02 00:25:27.920 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [handle_msgtype3C] interval=0:00:00.227661  td=0:01:00   self.lastSendOfDownloadEprom(UTC)=2024-08-01 23:25:27.692837    timenow(UTC)=2024-08-01 23:25:27.920507
2024-08-02 00:25:28.582 DEBUG (MainThread) [custom_components.visonic.pyvisonic] [_processReceivedMessage] Received Packet 0d b0 03 51 06 ff 08 ff 01 24 a7 43 dc 0a
2024-08-02 00:26:58.550 WARNING (MainThread) [custom_components.visonic.pyvisonic] [Controller] ********************** Download Timer has Expired, Download has taken too long *********************
2024-08-02 00:26:58.550 WARNING (MainThread) [custom_components.visonic.pyvisonic] [Controller] ************************************* Going to standard mode ***************************************
```

**Comment by davesmeghead** [comment]
I believe that this is fixed in Integration release 0.9.9.5.
